components/ui-api-layer/docs/usage-kind-resources.md

```diff
@@ -0,0 +1,12 @@
+# Query usageKindResources
+
+The query returns all instances of the resources specified by **usageKind** in the given **environment**. 
+The result is filtered by the metadata.ownerReference field. The result contains resources which does not contain **metadata.ownerReference**. 
```

Comment:
``The **metadata.ownerReference** field filters the result. It contains the resources which do not contain **metadata.ownerReference**. ``